scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.43.0] - 2020-06-15
+### Added
+- Python Client API, from @hdrodz97
```

Comment:
I think we should remove this, since this change log is specific to the scheduler directory.

Reply:
Ok.